web/src/types/dasd.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+type DASDDevice = {
+  id: string;
+  enabled: boolean;
+  deviceName: string;
+  formatted: boolean;
+  diag: boolean;
+  status: string; // TODO: sync with rust when it switch to enum
```

Comment:
What does it mean?

Reply:
@jreidinger ?

Reply:
it is related to FIXME in rust code that wants to use enums for some data like `status`. So when it rust switch to enum, we should also change it here.